mw-webapp/src/component/header/Header.tsx

```diff
@@ -0,0 +1,31 @@
+import {NavLink} from "src/component/link/Link";
+import styles from "src/component/header/Header.module.scss";
+import {useAuth} from "src/utils/useAuth";
+import {Button} from "src/component/button/Button";
+import {pages} from "src/router/pages";
+
+const handleGoogleSignIn = useAuth();
+const buttonValue = "Sign in with Google";
+const linkValue = "Workflow";
+const logo = "master-way";
+
```

Comment:
1. We have conventions in docs folder. There are info about consts naming. In this case we  using UPPER_CASE 
(if you don't agree with conventions, write to group - we will discuss it)
2. const logo better to change LOGO_TEXT . Do you agree?
3. "master-way" -> "master's way"

Reply:
Done.